### PR TITLE
ROS2 Humble

### PR DESCRIPTION
**Pull request by 0nhc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,202 @@
...
+Automatically setup for terminal.
+```sh
+echo "source ~/colcon_ws/install/setup.bash" >> ~/.bashrc
+echo "export ROS_DOMAIN_ID=42" >> ~/.bashrc
+```
+ 
+### 1.2 Mini Pupper Setup
+**Ubuntu 22.04 + ROS2 Humble is required.** 
+ 
+Install BSP(Board Support Package) for MiniPupper: [mini_pupper_bsp](https://github.com/mangdangroboticsclub/mini_pupper_bsp)
```
Comment (CullenSUN) [inline README.md:47]: MiniPupper ==> Mini Pupper

File: mini_pupper_bringup/launch/bringup.launch.py [hunk 2]
```diff
@@ -79,5 +85,37 @@ def generate_launch_description():
                 "gait_config_path": gait_config,
                 "description_path": description_path
             }.items(),
-        )
+        ),
+
+        IncludeLaunchDescription(
+            PythonLaunchDescriptionSource(servo_bringup_launch_path),
+            launch_arguments={}.items(),
+            condition = IfCondition(hardware_connected),
```
Comment (CullenSUN) [inline mini_pupper_bringup/launch/bringup.launch.py:93]: let's remove space around equal sign 
reference https://stackoverflow.com/questions/24810771/python-pep8-conventions

File: mini_pupper_navigation/maps/mymap.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+image: mymap.pgm
```
Comment (CullenSUN) [inline mini_pupper_navigation/maps/mymap.yaml:1]: can we just name it map.pgm? 
mymap sounds like a small personal project

**Comment by Ericsii** [comment]
Please close this PR. Then I will resubmit some new PR for ROS Humble migration for @0nhc.

**Comment by Ericsii** [comment]
Please close this PR. Then I will resubmit some new PR for ROS Humble migration instead of @0nhc.

**Pull request closed without merging**